src/Umbraco.Web.Common/Templates/TemplateRenderer.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewEngines;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Core;
+using Umbraco.Core.Configuration.UmbracoSettings;
+using Umbraco.Core.Services;
+using Umbraco.Core.Strings;
+using Umbraco.Extensions;
+using Umbraco.Web.Models;
+using Umbraco.Web.Routing;
+using Umbraco.Web.Templates;
+
+namespace Umbraco.Web.Common.Templates
+{
+    /// <summary>
+    /// This is used purely for the RenderTemplate functionality in Umbraco
+    /// </summary>
+    /// <remarks>
+    /// This allows you to render an MVC template based purely off of a node id and an optional alttemplate id as string output.
+    /// </remarks>
+    internal class TemplateRenderer : ITemplateRenderer
```

Comment:
I assume this file was just moved? But i don't see it being deleted anywhere either so not sure if its changed?

Reply:
Deleted the old one